src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/DynamicViewExpansion.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using Microsoft.VisualStudio.Debugger.Clr;
+using Microsoft.VisualStudio.Debugger.Evaluation;
+using Microsoft.VisualStudio.Debugger.Evaluation.ClrCompilation;
+
+namespace Microsoft.CodeAnalysis.ExpressionEvaluator
+{
+    internal sealed class DynamicViewExpansion : Expansion
+    {
+        private const string DynamicFormatSpecifier = "dynamic";
+
+        internal static DynamicViewExpansion CreateExpansion(DkmInspectionContext inspectionContext, DkmClrValue value, Formatter formatter)
+        {
+            var type = value.Type.GetLmrType();
+            if (!(type.IsComObject() || type.IsIDynamicMetaObjectProvider()))
+            {
+                return null;
+            }
+
+            var proxyValue = value.InstantiateDynamicViewProxy(inspectionContext);
+            // InstantiateDynamicViewProxy may return null (if required assembly is missing, for instance).
+            if (proxyValue == null)
+            {
+                return null;
+            }
+
+            // Expansion is based on the 'DynamicMetaObjectProviderDebugView.Items' property.
+            var proxyType = proxyValue.Type;
+            var itemsMemberExpansion = RootHiddenExpansion.CreateExpansion(
```

Comment:
It does have the attribute, but in addition to hiding it, we **only** want to show the results of the Items property.